config/config_test.go

```diff
@@ -336,8 +336,8 @@ func Test_NewConfigWithArgs(t *testing.T) {
 			}
 
 			if !tt.wantErr {
-				if cmp.Equal(cfg, tt.want, cmp.AllowUnexported(listFlag{}, pluginFlag{}, defaultFiltersFlags{}, mapFlags{}), cmpopts.IgnoreUnexported(Config{})) == false {
-					t.Errorf("config.NewConfig() got vs. want:\n%v", cmp.Diff(cfg, tt.want, cmp.AllowUnexported(listFlag{}, pluginFlag{}, defaultFiltersFlags{}, mapFlags{}), cmpopts.IgnoreUnexported(Config{})))
+				if cmp.Equal(cfg, tt.want, cmp.AllowUnexported(listFlag{}, pluginFlag{}, defaultFiltersFlags{}, mapFlags{}), cmpopts.IgnoreUnexported(Config{}), cmpopts.IgnoreFields(Config{}, "Flags")) == false {
```

Comment:
@szuecs The failing test has been addressed. Before this PR, the value of `Config.flags` was not considered for comparison during these tests because of `cmpopts.IgnoreUnexported(Config{})`. Added `cmpopts.IgnoreFields(Config{}, "Flags")` to re-create the behavior of ignoring the field.